vulnerabilities/templates/vulnerabilities.html

```diff
@@ -18,7 +18,17 @@
                 <div>
                     {{ page_obj.paginator.count|intcomma }} results
                 </div>
-              {% if is_paginated %}
+                <div class="is-flex is-justify-content-center mb-2">
+                    <div class="select is-small">
+                        <select id="itemsPerPage" onchange="changeItemsPerPage(this.value)">
+                            <option value="20" {% if page_obj.paginator.per_page == 20 %}selected{% endif %}>20 per page</option>
+                            <option value="50" {% if page_obj.paginator.per_page == 50 %}selected{% endif %}>50 per page</option>
+                            <option value="100" {% if page_obj.paginator.per_page == 100 %}selected{% endif %}>100 per page</option>
+                            <option value="200" {% if page_obj.paginator.per_page == 200 %}selected{% endif %}>200 per page</option>
+                        </select>
+                    </div>
+                </div>
+                {% if is_paginated %}
```

Comment:
Duplicated code.

Reply:
```
    PAGE_CHOICES = [
        ("20", "20 per page"),
        ("50", "50 per page"),
        ("100", "100 per page"),
    ]

```
Now In the latest commit it is address via forms.py.